package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@vesselapi/integrations",
-  "version": "1.0.67",
+  "version": "1.0.68-rc.0",
```

Comment:
oh hmm did someone accidentally merge a release candidate?

Reply:
Ah no, I was just doing it so that the core PR would build. But I'll change everything to a release version before merging